src/Form.vue

```diff
@@ -204,13 +204,20 @@ export default {
       switch (question.type) {
         case "list":
         case "rawlist":
+          if (question._default === undefined) {
+            question.answer = undefined;
```

Comment:
You're inside a method called `getInitialAnswer()`. Don't set the `question.answer` here -- return the answer.

I believe this entire case (`list` and `rawlist`) can be written like this:
```js
if (question._choices.length === 0 || question._default === undefined) {
    this.setInvalid(question, NOT_ANSWERED);
    return;
}
if (typeof question._default === "number" && question._default < question._choices.length) {
    index = question._default;
}
if (index < 0) {
    index = question._choices.findIndex(choice => {
        if (question._default) {
            return choice.value === question._default;
        }
    });
}
if (index < 0) {
    this.setInvalid(question, NOT_ANSWERED);
    return;
} else {
    return question._choices[index].value;
}
```

... but please verify

Reply:
done